lib/screens/v2/widget_instance/subway_status.ex

```diff
@@ -135,7 +135,7 @@ defmodule Screens.V2.WidgetInstance.SubwayStatus do
     def audio_sort_key(_instance), do: [2]
 
     def audio_valid_candidate?(%{screen: %Screen{app_params: %app{}}})
-        when app in [PreFare, GlEink],
+        when app in [BusEink, GlEink, PreFare],
```

Comment:
Same wondering here as above.

Reply:
This is likely done for the same reason as above. Something to think about here is whether it's not a bad experience to hear subway audio at a bus stop. That's the only thing I could see being an issue and even that is reaching.